kolibri/plugins/coach/assets/src/views/lessons/LessonSummaryPage/index.vue

```diff
@@ -145,7 +144,6 @@
       const workingResourcesBackup = [...(this.$store.state.lessonSummary.workingResources || [])];
 
       return {
-        PageNames,
```

Comment:
I think the diff can just be reduced to deleting this and making sure we're not referencing `this.PageNames` anywhere in the component.

As I suspect that linting saying this was undefined might have driven the inclusion here initially.